### DISCUSSION TITLE
The future of Filters

### DISCUSSION THREAD

**Discussion by sarahsporck** [software-mansion/react-native-svg#2100] [General]
Hello there! 👋🏼 
I'am a developer using this library in several projects for quite some time now and honestly am also quite happy on how it works. Recently, we started to moved from using the react-native-asset-plugin to the react-native-svg-transformer, which meant that all svgs would be rendered using this library. That's when I stumbled upon the filters issue, which has been open since 2019 already and seems to be stale now for about two years. 
Although we don't heavily require filters, I was wondering what's the plan here. There have been several people motivated to implement the feature, some even submitted a bounty, but the response has been awfully quiet (no offense, open-source requires it's time). However, I am curious on whether you still plan to implement filters. What's missing here? Is it a matter of planning? I'm certainly far from being an expert, but I guess you could start off by implementing just one or two filters. 
Feel free to critique or just dump you're thought's 🧠 :)
Looking forward to a constructive discussion :)
Cheers - Sarah

**Comment by bohdanprog** [comment]
Hello @sarahsporck,
We appreciate your usage of our library. 
Currently, we are working on integrating those functionalities into our library. 
Some of them have already been added, as detailed [here](https://github.com/software-mansion/react-native-svg/blob/main/USAGE.md#filters).
You can track our progress [here](https://github.com/software-mansion/react-native-svg/issues/2362).
